automation_tools/__init__.py

```diff
@@ -1003,21 +1003,19 @@ def cdn_install(run_katello_installer=True):
 
     The following environment variables affect this command:
 
-    RHN_USERNAME
-        Red Hat Network username.
-    RHN_PASSWORD
-        Red Hat Network password.
-    RHN_POOLID
-        Optional. Red Hat Network pool ID. Determines what software will be
-        available from RHN.
     ADMIN_PASSWORD
         Optional, defaults to 'changeme'. Foreman admin password.
+    SATELLITE_VERSION
+        Satellite version.
 
     """
     admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
 
     # Install required packages for the installation
-    run('yum install -y katello')
+    if os.environ.get('SATELLITE_VERSION') == '6.2':
```

Comment:
here it is similar with donwstream_install:
product_install -> downstream_install (requires SATELLITE_VERSION)
product_install -> cdn_install (requires SATELLITE_VERSION)

I better switch logic here if SATELLITE_VERSION not defined or 6.2 -> satellite
Now it is if not defined is the same as 6.1 -> katello